minio-operator.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: minio-operator
+  version: 6.0.2
+  epoch: 0
+  description: Minio Operator creates/configures/manages Minio on Kubernetes
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/minio/operator
+      tag: v${{package.version}}
+      expected-commit: 15c388bd4ce0a03084f1b3762f16a65679b83423
+
+  - uses: go/bump
+    with:
+      deps: github.com/docker/docker@v27.1.1
+      modroot: .
+
+  - uses: go/build
+    with:
+      modroot: ./cmd/operator
+      packages: .
+      output: minio-operator
+      ldflags: -s -w -X github.com/minio/operator/pkg.ReleaseTag=${{package.full-version}} -X github.com/minio/operator/pkg.Version=${{package.full-version}} -X github.com/minio/operator/pkg.ShortCommitID=$(git rev-parse HEAD)
+
+  - uses: strip
+
+  - runs: |
+      mkdir ${{targets.destdir}}/licenses
+      cp CREDITS LICENSE ${{targets.destdir}}/licenses/
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: compatibility symlinks package for minio-operator Dockerfile
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/
+          ln -sf /usr/bin/minio-operator ${{targets.contextdir}}/minio-operator
```

Comment:
Could this be part of the main package to simplify things a bit?

Reply:
we have been using compat packages for these symlinks as you know, its ok for me but I'm not sure we should break the standard here?

Reply:
all good, works for me!